api.py

```diff
@@ -0,0 +1,129 @@
+# Start the api server
+# python api.py
+# load a book to the database, book in folder ./books
+# curl -X POST -H "Content-Type: application/json" -d '{"filename": "new_book.epub"}' http://127.0.0.1:5448/api/load_book
+
+import datetime
+from flask import Flask, request, jsonify
+from tqdm import tqdm
+import secrets
+import os
+from ebooklib import epub
+import ebooklib
+from bleach import clean, sanitizer
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
```

Comment:
you can `import * from app` to get the app, db, database models and functions so you don't have to duplicate it here.

Reply:
The common part (like Book, Chapter) should be extracted from app, so api can import, rather than importing from app

Reply:
It's fine